src/styles/blocks/stop-watch.scss

```
@@ -0,0 +1,78 @@
+.stop-watch {
+  display: block;
+
+  padding: 50px;
+
+  background: #eae7dc;
```

don't remember to create variables for colors

```suggestion
  background: $c-color;
```